estimate-web/src/components/UserStoryCommentList.less

```diff
@@ -0,0 +1,23 @@
+@import "../resources/Measurements.less";
+@import "../resources/Colors.less";
+
+.user-story-comments {
```

Comment:
Seems to share some things with `user-story-comment-editor`, might be worth consolidating